Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -2,94 +2,137 @@
 {
     public static class ArrayOperations
     {
-        /// <summary>
-        /// Sort the array in ascending order.
-        /// If array empty or null- don't do anything.
-        /// </summary>
-        /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (array == null || array.Length == 0)
+            {
+                return;
+            }
+            int result = 0;
+            for (int i = 0; i < array.Length - 1; i++)
+            {
+                for (int j = i + 1; j < array.Length; j++)
+                {
+                    if (array[j] < array[i])
+                    {
+                        result = array[i];
+                        array[i] = array[j];
+                        array[j] = result;
+                    }
+                }
+            }
         }
-
-        /// <summary>
-        /// Reverse the array elements, first being last and so on.
-        /// If array empty or null- don't do anything.
-        /// </summary>
-        /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            if (array == null || array.Length == 0)
+            {
+                return;
+            }
+            int revArray = array.Length;
+            for (int i = 0; i < revArray / 2; i++)
+            {
+                int temp = array[i];
```

Comment:
Reverse algorithm and BubbleSort actually share one similar part- element swap :)
You can create Swap method and reuse it in both functions.